sonar-python-plugin/src/main/java/org/sonar/plugins/python/IpynbNotebookParser.java

```diff
@@ -171,6 +172,10 @@ private boolean parseSourceMultilineString(JsonParser jParser, JsonToken jsonTok
       aggregatedSource.append("\n");
       previousLen = previousLen + line.length() + 2;
```

Comment:
I do not understand why the first line of the multiline cell should have 1 more character

Reply:
The extra character is need to align the behavior between the multiline strings cells and the array cells. The location starts on the first `"`, before the Python code. The function did not account for that situation previously.